settings.gradle.kts

```diff
@@ -3,3 +3,29 @@ rootProject.name = "rep-bot"
 plugins {
     id("org.gradle.toolchains.foojay-resolver-convention") version("0.6.0")
 }
+
+dependencyResolutionManagement {
+    versionCatalogs {
```

Comment:
I'm not sure, is renovate able to handle this?

Reply:
Yes it is and that is the nice stuff! xD

Reply:
Renovate even recognises the shared versions and groups them into a single update pr

Reply:
Can be seen here: https://github.com/rainbowdashlabs/krile/issues/5